packages/blocks/src/api/raw-handling/shortcode-converter.js

```diff
@@ -91,17 +92,22 @@ function segmentHTMLToShortcodeBlock(
 			( schema ) => schema.shortcode( match.shortcode.attrs, match )
 		);
 
-		const block = createBlock(
+		const transformationBlockType = {
+			...getBlockType( transformation.blockName ),
+			attributes: transformation.attributes,
+		};
+
+		let block = createBlock(
```

Comment:
To be honest, I found that the behavior of the shortcode converter is very weird. It builds this kind of intermediary "block type" to just to be able to used the `getBlockAttributes` function. I think a dedicated custom parser would have been better than trying to fit this with the code used for block parsing.

That said, I'm not going down that rabit hole for now, I'm just making sure it works like before.